Tests/StencilSwiftKitTests/StringFiltersTests.swift

```diff
@@ -140,11 +140,12 @@ extension StringFiltersTests {
 
   func testRemoveNewlines_WithTrue() throws {
     let expectations = [
-      "test": "test",
-      "  \n test": "test",
-      "test  \n ": "test",
-      "test\n \ntest": "testtest",
-      "\n test\n \ntest \n ": "testtest"
+      "test1": "test1",
+      "  \n test2": "test2",
+      "test3  \n ": "test3",
+      "test4\n \ntest": "test4test",
+      "\n test5\n \ntest test \n ": "test5test test",
+      "test6\ntest": "test6test"
```

Comment:
Still, I like that the examples we give in the MD documentation are used in the tests as well, if only to be sure that we are consistent with what we state in the doc and that those cases are indeed checked